Sample/View Controllers/Alert/AlertViewController.swift

```diff
@@ -65,4 +65,10 @@ class AlertViewController: UIViewController, PanModalPresentable {
     var isUserInteractionEnabled: Bool {
         return true
     }
+
+    func makeAnimator() -> UIViewPropertyAnimator {
+        let animator = UIViewPropertyAnimator(duration: 0.5, dampingRatio: 0.7, animations: nil)
```

Comment:
I think notifications are a great place for springy animations, and provided an example of a custom animator there.